scripts/publish-sentry-release.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -o errexit -o pipefail
+
+# This script packages our JS source files and sourcemaps as a Sentry release.
+# https://docs.sentry.io/product/releases/
+
+source ./scripts/common.sh
+
+build_dir="public"
+source_dir="assets"
+
+# Make a folder to contain the source files and maps we upload to Sentry.
+release_dir="sentry-release"
+rm -rf "${release_dir}"
```

Comment:
It doesn't quite seem right to create a folder immediately after blowing it away. Also, is there any reason why the release should be in the current directory? Could we just write the Sentry stuff to tmp?
```
# Just put the sentry release into a temp folder,
# so no need to add to .gitignore or delete on publish.
readonly release_dir=$(mktemp -d -t "sentry-release")
...
```